src/APU.h

```diff
@@ -46,13 +58,15 @@ class APU {
 	uint16_t readPos = 0;
 	uint16_t writePos = 0;
 	
+	// TODO: Simplify apu data
```

Comment:
This is pretty important, the APU is getting quite messy. I wonder if each channel should have a public struct maybe? That would probably be a simpler, more readable, and less error prone way of keeping track of the data. Not sure if its slower but readability and organization are probably more important.